Description: Rename executable program to avoid namespace clash.
 .
 The executable program is to exist alongside the application's importable
 library package. So it needs to have a different filename in that directory.
 .
 Created to work with “entry points” feature of Python's Distutils.
Author: Ben Finney <bignose@debian.org>
Last-Update: 2023-02-14

diff --git old/setup.py new/setup.py
--- old/setup.py
+++ new/setup.py
@@ -61,5 +61,5 @@ setup(
     include_package_data=True,
     description="Building newsfiles for your project.",
     long_description=open("README.rst").read(),
-    entry_points={"console_scripts": ["towncrier = towncrier._shell:cli"]},
+    entry_points={"console_scripts": ["execute-towncrier = towncrier._shell:cli"]},
 )

Local variables:
coding: utf-8
mode: diff
time-stamp-format: "%:y-%02m-%02d"
time-stamp-start: "^Last-Update:[ 	]+"
time-stamp-end: "$"
time-stamp-line-limit: 20
End:
vim: fileencoding=utf-8 filetype=diff :
